src/content/windowEventRecorder.js

```diff
@@ -131,33 +132,30 @@ if (!window.loadsterRecorderScriptsLoaded) {
     }
   };
 
-  function getCssSelectors (element, frameSelector) {
+  function getCssSelectors(element, frameSelector) {
     const idFilter = (value) => !filters.idName.some(p => p.test(value));
-    const classFilter = (value) => !filters.idName.some(p => p.test(value));
+    const classFilter = (value) => !filters.className.some(p => p.test(value));
     const tagFilter = (name) => !filters.tagName.some(p => p.test(name));
```

Comment:
Oh, thanks. I looked at this so many times and never saw it.

Reply:
Yeah, I didn't either until I started trying to isolate different selector types and wondered why I didn't get any!